src/main/java/org/maveric/quarkus/panache/exceptionHandler/SavingsAccountDetailsNotFoundExceptionMapper.java

```diff
@@ -11,8 +10,9 @@ public class SavingsAccountDetailsNotFoundExceptionMapper implements ExceptionMa
     @Override
     public Response toResponse(SavingsAccountDetailsNotFoundException e) {
         ErrorResponse.ErrorMessage errorMessages =
-                new ErrorResponse.ErrorMessage(e.getMessage());
 
+                new ErrorResponse.ErrorMessage("/a/b", e.getMessage());
```

Comment:
What is /a/b?

Reply:
sorry for testing purpose i gave that , i will change it now